canonical-web-and-design/maas-ui
# fix(legacy): stop setting gateway on gateway-less subnets (#1860)

## Done

Unify the behavior for both standard and relayed networks, populating the placeholder with the suggested gateway without setting the IP when the subnet does not have one configured.

## QA

- Configure a subnet without gateway
- Go to related VLAN and configure DHCP
- Verify that the gateway field has remained unchanged

## Fixes

Fixes: #1860 .

## Launchpad issue

lp#1903586

## Backports

2.8 and 2.7 are affected too
